Yoast/wordpress-seo
# P3 123 add tests for semrush redux

## Context


* In this PR we add tests to make sure the redux code for semrush works as expected

## Summary


This PR can be summarized in the following changelog entry:

* Adds tests for all the available semrush redux actions, reducers and selectors

## Relevant technical choices:

* Removed the unused parameters for the function `setNoResultsFound`. Discussed with @jcomack that this could be done in this PR and a separate issue is unnecessary we think.
* I have chosen to write a separate test for each redux function with applicable test data (the one it receives in production) so when something breaks we know exactly which function it is.

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch and build just in case
* Run `yarn test`
* Ensure the tests pass

## UI changes

## Documentation

## Quality assurance
